pkg/reconciler/subscription/controller.go

```diff
@@ -63,5 +64,17 @@ func NewController(
 	r.channelableTracker = duck.NewListableTracker(ctx, channelable.Get, impl.EnqueueKey, controller.GetTrackerLease(ctx))
 	r.destinationResolver = resolver.NewURIResolver(ctx, impl.EnqueueKey)
 
+	// Track changes to Channels.
```

Comment:
Can you add more details on why the L64 tracker is not sufficient. Want to make sure that this doesn't get optimized away in the future :) As in the deets of the bug we saw :)

Reply:
I could add the issue I filed for this and more comments.

Reply:
Added.